llama-index-experimental/pyproject.toml

```diff
@@ -31,6 +31,7 @@ version = "0.3.0"
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.11.0"
 pandas = "*"
+duckdb = "1.0.0"
```

Comment:
rather than pinning a specific version, we could do something like `ducked = "^1.0.0"`